signing.go

```diff
@@ -272,3 +285,31 @@ func (obj JSONWebSignature) VerifyMulti(verificationKey interface{}) (int, Signa
 
 	return -1, Signature{}, nil, ErrCryptoFailure
 }
+
+func verifyCerts(data *rawHeader) error {
+	if len(data.X5c) == 0 {
+		return nil
```

Comment:
I would return an error here, then only call this function if `len(data.X5c) > 0`. Otherwise someone calling this accidentally with an empty chain might not get what they want.